### PR TITLE
use gpg instead of gpg2

### PR DESCRIPTION
**Pull request by codifryed**
Hi there :wave:,

TLDR:
This pretty simple PR would use the `gpg` command instead of `gpg2`. 

Reasoning:
Most, if not all distros have deprecated gpg v1 and the standard `gpg` package is now v2. gpg2 is currently a "dummy transitional package." and many distros do not install it by default, meaning extra user interaction is currently required to enable gpg signature validation of appimages. 

Research:
I checked quite a few distros and even the older supported ones like ubuntu 18.04 and centos 7 are using gpg v2 as the standard installed package, meaning I don't see any reason to continue using gpg2. 

This is somewhat in relation to https://github.com/AppImage/AppImageUpdate/issues/97

### CODE REVIEW

**Comment by probonopd** [comment]
Thanks for pointing this out. There is a difference between gpg v2 and gpg2? Talk about confusing...

**Comment by codifryed** [comment]
:+1: It is kinda confusing. Perhaps a better way to explain it:

Back in the day:
    `gpg` was gpg v1
    `gpg2` was gpg v2
     simple enough.

Then they upgraded the `gpg` package to gpg v2, deprecated and moved v1 to `gpg1` and kept `gpg2` as a transitional package so nothing broke.

Now:
  `gpg` == `gpg2` == gpg v2

since the `gpg` package is installed by default and `gpg2` is not....

Ofc this PR could be a one-liner, I just thought perhaps for consistency & clarity to change all the references. :shrug:

**Comment by TheAssassin** [comment]
Please add this as a fallback. `gpg2` should be preferred if available, otherwise `gpg` can be used.

File: src/updater.cpp [hunk 1]
```diff
@@ -970,7 +973,7 @@ namespace appimage {
                     trim(line, '\n');
                     trim(line);
 
-                    d->issueStatusMessage(std::string("gpg2: ") + line);
+                    d->issueStatusMessage(std::string("gpg: ") + line);
```
Comment (TheAssassin) [inline src/updater.cpp:976]: You should reuse `gpgPath` here. This'll allow users to see which binary was actually used.

**Review by TheAssassin — CHANGES_REQUESTED** [review]
One small change request, otherwise looks really good, thanks!

File: src/updater.cpp (hunk 1 above)
Comment (codifryed) [reply src/updater.cpp]: ah,  good idea. :smile:

**Comment by TheAssassin** [comment]
Since I'll have to rebase my PR anyway due to https://github.com/AppImage/AppImageUpdate/commit/eb3d09d2a21d84e8fbeacf830b357895d48e70ab, I'll merge this PR first.

**Review by TheAssassin — APPROVED** [review]
